docs/src/dev/developer/development-environment.asciidoc

```diff
@@ -183,6 +197,11 @@ authentication information on the ~/.npmrc file. The easiest way to do that is t
 must be done only once, as the auth token doesn't have an expiration date and it's stored on your file system. If
 this account is newly created then request that a PMC member add your account to the "gremlin" package on npm.
 
+Deploying Docker images to link:https://hub.docker.com/[Docker Hub] requires authentication information in the
```

Comment:
Please add a list item here in the "Release & Promote" section so that we remember to publish docker images:

http://tinkerpop.apache.org/docs/current/dev/developer/#_release_promote

Reply:
"Release Environment" should also make mention that the person doing the deployment needs a dockerhub account and needs to request addition from a PMC member to the TinkerPop organization.